x/clp/keeper/executors.go

```diff
@@ -36,10 +36,11 @@ func (k Keeper) CreatePool(ctx sdk.Context, poolUints sdk.Uint, msg *types.MsgCr
 		return nil, types.ErrBalanceNotAvailable
 	}
 
-	pool, err := types.NewPool(msg.ExternalAsset, msg.NativeAssetAmount, msg.ExternalAssetAmount, poolUints)
-	if err != nil {
-		return nil, sdkerrors.Wrap(types.ErrUnableToCreatePool, err.Error())
-	}
+	// (testing) this part can not be triggered, because newPool always return nil err LOL.
+	pool := types.NewPool(msg.ExternalAsset, msg.NativeAssetAmount, msg.ExternalAssetAmount, poolUints)
+	//if err != nil {
```

Comment:
This should be cleaned up comment wise, the above comment doesn't apply if this is commented out, or better just removed.